Specs/DateTime/Chinese/DateTimeModel.json

```diff
@@ -6905,5 +6905,61 @@
         }
       }
     ]
+  },
+  {
+    "Input": "帮我预订2016年12月15日晚上7:00到8:00的会议",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T10:20:00"
+    },
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "2016年12月15日晚上7:00到8:00",
+        "Start": 4,
+        "End": 25,
+        "TypeName": "datetimeV2.datetimerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(2016-12-15T19:00,2016-12-15T20:00,PT1H)",
```

Comment:
Why does this one still has the 00s?

Reply:
because the time in the text also has the :00. 
Standard implementation follows the same logic: 
8 pm -> T20 
8:00 pm -> T20:00.